pkg/report/testdata/linux/report/379

```diff
@@ -0,0 +1,28 @@
+TITLE: INFO: task hung in __switch_to
```

Comment:
After I hit merge it occurred to me that this should be attributed to "in exit_aio". Otherwise all task hangs will be detected as "in __switch_to". Please add more skip patterns for "task hung" reports.

Reply:
Good point, I'll go through our logs and see if there's any other interesting skip patterns we should add